src/sass/form-updates.scss

```diff
@@ -0,0 +1,71 @@
+.has-feedback {
```

Comment:
I just want to see how you write css. Please create several classes with semantic names and without nesting and use them instead of overrided bootstrap.